bindings/c/include/opendal.h

```diff
@@ -132,8 +134,7 @@ typedef const struct BlockingOperator *opendal_operator_ptr;
  the reallocation of the Vector.
  */
 typedef struct opendal_bytes {
-  const uint8_t *data;
-  uintptr_t len;
+  const struct Vec_u8 *v;
```

Comment:
Maybe we don't need to expose `Vec_u8` directly, a point and a length maybe enough?